<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PPassivesWindow</class>
 <widget class="QMdiSubWindow" name="PPassivesWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Passive and Ascendancy Skills</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="_ContentWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>9</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="_TopLayout">
        <item>
         <layout class="QGridLayout" name="_SummaryGrid">
          <item row="3" column="3">
           <widget class="QLineEdit" name="_PassAllocEdit">
            <property name="minimumSize">
             <size>
              <width>25</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>115</string>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QLineEdit" name="_TotalEdit">
            <property name="minimumSize">
             <size>
              <width>25</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>115</string>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QLabel" name="_AllocLbl">
            <property name="text">
             <string>Alloc</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="_PassivesLbl">
            <property name="text">
             <string>Passive skill points </string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLabel" name="_AscendLbl">
            <property name="text">
             <string>Ascendancy points</string>
            </property>
           </widget>
          </item>
          <item row="2" column="4">
           <widget class="QLabel" name="_FromLevelLbl">
            <property name="text">
             <string>Level</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="5">
           <widget class="QLabel" name="_FromQuestsLbl">
            <property name="text">
             <string>Quests</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QLineEdit" name="_AscTotalEdit">
            <property name="minimumSize">
             <size>
              <width>25</width>
              <height>0</height>
             </size>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="3">
           <widget class="QLineEdit" name="_AscAllocEdit">
            <property name="minimumSize">
             <size>
              <width>25</width>
              <height>0</height>
             </size>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="_TotalPassLbl">
            <property name="text">
             <string>Total</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="3" column="4">
           <widget class="QLineEdit" name="_PassFromLvlEdit">
            <property name="minimumSize">
             <size>
              <width>25</width>
              <height>0</height>
             </size>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="5">
           <widget class="QLineEdit" name="_PassFromQuestsEdit">
            <property name="minimumSize">
             <size>
              <width>25</width>
              <height>0</height>
             </size>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="_RefreshBtn">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="PoePal.qrc">
            <normaloff>:/PoePal/Resources/32x32/arrow_refresh.png</normaloff>:/PoePal/Resources/32x32/arrow_refresh.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="_TopSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="_QuestsBox">
        <property name="title">
         <string>Passives From Quests</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QTreeWidget" name="_QuestsTree">
           <property name="indentation">
            <number>20</number>
           </property>
           <attribute name="headerVisible">
            <bool>false</bool>
           </attribute>
           <column>
            <property name="text">
             <string>1</string>
            </property>
           </column>
           <item>
            <property name="text">
             <string>Act 1 (0/3)</string>
            </property>
            <property name="checkState">
             <enum>Unchecked</enum>
            </property>
            <property name="flags">
             <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsEnabled|ItemIsTristate</set>
            </property>
            <item>
             <property name="text">
              <string>The Marooned Mariner</string>
             </property>
             <property name="checkState">
              <enum>Unchecked</enum>
             </property>
             <property name="flags">
              <set>ItemIsSelectable|ItemIsEnabled</set>
             </property>
            </item>
            <item>
             <property name="text">
              <string>The Dweller of the Deep</string>
             </property>
             <property name="checkState">
              <enum>Unchecked</enum>
             </property>
             <property name="flags">
              <set>ItemIsSelectable|ItemIsEnabled</set>
             </property>
            </item>
            <item>
             <property name="text">
              <string>The Way Forward</string>
             </property>
             <property name="checkState">
              <enum>Unchecked</enum>
             </property>
             <property name="flags">
              <set>ItemIsSelectable|ItemIsEnabled</set>
             </property>
            </item>
           </item>
           <item>
            <property name="text">
             <string>Act 2 (0/2)</string>
            </property>
            <property name="checkState">
             <enum>Unchecked</enum>
            </property>
            <property name="flags">
             <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsEnabled|ItemIsTristate</set>
            </property>
            <item>
             <property name="text">
              <string>Deal with the Bandits (2)</string>
             </property>
             <property name="checkState">
              <enum>Unchecked</enum>
             </property>
             <property name="flags">
              <set>ItemIsSelectable|ItemIsEnabled</set>
             </property>
            </item>
           </item>
           <item>
            <property name="text">
             <string>Act 3 (0/2)</string>
            </property>
            <property name="checkState">
             <enum>Unchecked</enum>
            </property>
            <property name="flags">
             <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled|ItemIsTristate</set>
            </property>
            <item>
             <property name="text">
              <string>Victario's Secrets</string>
             </property>
             <property name="checkState">
              <enum>Unchecked</enum>
             </property>
             <property name="flags">
              <set>ItemIsSelectable|ItemIsEnabled</set>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Piety's Pets</string>
             </property>
             <property name="checkState">
              <enum>Unchecked</enum>
             </property>
             <property name="flags">
              <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsEnabled</set>
             </property>
            </item>
           </item>
           <item>
            <property name="text">
             <string>Act 4 (0/2)</string>
            </property>
            <property name="checkState">
             <enum>Unchecked</enum>
            </property>
            <property name="flags">
             <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled|ItemIsTristate</set>
            </property>
            <item>
             <property name="text">
              <string>An Indomitable Spirit</string>
             </property>
             <property name="checkState">
              <enum>Unchecked</enum>
             </property>
            </item>
           </item>
           <item>
            <property name="text">
             <string>Act 5 (0/2)</string>
            </property>
            <property name="checkState">
             <enum>Unchecked</enum>
            </property>
            <property name="flags">
             <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled|ItemIsTristate</set>
            </property>
            <item>
             <property name="text">
              <string>In Service to Science</string>
             </property>
             <property name="checkState">
              <enum>Unchecked</enum>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Kitava's Torments</string>
             </property>
             <property name="checkState">
              <enum>Unchecked</enum>
             </property>
            </item>
           </item>
           <item>
            <property name="text">
             <string>Act 6 (0/3)</string>
            </property>
            <property name="checkState">
             <enum>Unchecked</enum>
            </property>
            <property name="flags">
             <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled|ItemIsTristate</set>
            </property>
            <item>
             <property name="text">
              <string>The Father of War</string>
             </property>
             <property name="checkState">
              <enum>Unchecked</enum>
             </property>
            </item>
            <item>
             <property name="text">
              <string>The Puppet Mistress</string>
             </property>
             <property name="checkState">
              <enum>Unchecked</enum>
             </property>
            </item>
            <item>
             <property name="text">
              <string>The Cloven One</string>
             </property>
             <property name="checkState">
              <enum>Unchecked</enum>
             </property>
            </item>
           </item>
           <item>
            <property name="text">
             <string>Act 7 (0/3)</string>
            </property>
            <property name="checkState">
             <enum>Unchecked</enum>
            </property>
            <property name="flags">
             <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled|ItemIsTristate</set>
            </property>
            <item>
             <property name="text">
              <string>The Master of a Million Faces</string>
             </property>
             <property name="checkState">
              <enum>Unchecked</enum>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Kishara's Star</string>
             </property>
             <property name="checkState">
              <enum>Unchecked</enum>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Queen of Despair</string>
             </property>
             <property name="checkState">
              <enum>Unchecked</enum>
             </property>
            </item>
           </item>
           <item>
            <property name="text">
             <string>Act 8 (0/3)</string>
            </property>
            <property name="checkState">
             <enum>Unchecked</enum>
            </property>
            <property name="flags">
             <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled|ItemIsTristate</set>
            </property>
            <item>
             <property name="text">
              <string>Reflection of Terror</string>
             </property>
             <property name="checkState">
              <enum>Unchecked</enum>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Love is Dead</string>
             </property>
             <property name="checkState">
              <enum>Unchecked</enum>
             </property>
            </item>
            <item>
             <property name="text">
              <string>The Gemling Legion</string>
             </property>
             <property name="checkState">
              <enum>Unchecked</enum>
             </property>
            </item>
           </item>
           <item>
            <property name="text">
             <string>Act 9 (0/2)</string>
            </property>
            <property name="checkState">
             <enum>Unchecked</enum>
            </property>
            <property name="flags">
             <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled|ItemIsTristate</set>
            </property>
            <item>
             <property name="text">
              <string>The Ruler of Highgate</string>
             </property>
             <property name="checkState">
              <enum>Unchecked</enum>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Queen of the Sands</string>
             </property>
             <property name="checkState">
              <enum>Unchecked</enum>
             </property>
            </item>
           </item>
           <item>
            <property name="text">
             <string>Act 10 (0/2)</string>
            </property>
            <property name="checkState">
             <enum>Unchecked</enum>
            </property>
            <property name="flags">
             <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled|ItemIsTristate</set>
            </property>
            <item>
             <property name="text">
              <string>Vilenta's Vengeance</string>
             </property>
             <property name="checkState">
              <enum>Unchecked</enum>
             </property>
            </item>
            <item>
             <property name="text">
              <string>An End to Hunger (2)</string>
             </property>
             <property name="checkState">
              <enum>Unchecked</enum>
             </property>
            </item>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QMdiSubWindow</class>
   <extends>QWidget</extends>
   <header>qmdisubwindow.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="PoePal.qrc"/>
 </resources>
 <connections/>
</ui>
